src/material_hasher/hasher/utils/symmetry.py

```diff
@@ -1,11 +1,56 @@
 # Copyright 2025 Entalpic
 from shutil import which
 
+import moyopy
 from monty.tempfile import ScratchDir
+from moyopy.interface import MoyoAdapter
 from pymatgen.analysis.structure_analyzer import SpacegroupAnalyzer
 from pymatgen.core import Structure
 
 
+class MoyoSymmetry:
+    """
+    Moyo symmetry using Moyo library
+
+    Parameters
+    ----------
+    symprec : float, optional
+        Symmetry precision tollerance. Defaults to 1e-4.
+    angle_tolerance : float, optional
+        Angle tolerance. Defaults to None.
+    setting : str, optional
+        Setting. Defaults to None.
+    """
+
+    def __init__(
+        self, symprec: float = 1e-4, angle_tolerance: float = None, setting: str = None
```

Comment:
```suggestion
        self, symprec: float = 1e-4, angle_tolerance: float | None = None, setting: str | None = None
```

This is the proper type annotation to be used in case of optional arguments in the function signature. The first option (e.g `some_value: float = None`) is problematic because it tells type checkers that value is always a float, but then contradicts this by setting a default value of None. This would cause type checkers like mypy or pyright (VSCode) to flag an error since you're assigning a None value to something that's supposed to be a float.

Reply:
if the Python version is below 3.10, you'll need to use the `Optional` type from the `typing` module 

```python
from typing import Optional

def function(value: Optional[float] = None):
    ...
```

Reply:
Thanks for spotting it!